.pylintrc

```diff
@@ -1,17 +1,18 @@
 [MESSAGES CONTROL]
 # For all codes, run 'pylint --list-msgs' or go to 'http://pylint-messages.wikidot.com/all-codes'
 disable=
-    invalid-name,
+    cyclic-import,
     fixme,
+    import-outside-toplevel,
+    invalid-name,
     missing-docstring,
-    too-many-arguments,
+    raise-missing-from,
```

Comment:
This should not be forced: E0012: Bad option value 'bad-continuation' (bad-option-value)